### PR TITLE
Remove Themes from TraitsUI

### PR DESCRIPTION
**Pull request by corranwebster**
This mostly removes Themes from TraitsUI.  Minimal theming support is still needed for dock window themes, but that should be moved to the dock codebase itself as a separate PR.

There one a regression in this PR, which is that the VFold group is no longer supported on Wx.  It was implemented completely as part of the theming codebase, and had an idiosyncratic look and feel.  The VFold is rarely used in the enthought codebase (certaining never in ETS) so I think this regression is acceptable for now.

~Also, ScrubberEditor seems to be broken, but that may be an independent issue.~ It was requiring some of the image slicing code.  Now that that has been restored, it works.

### CODE REVIEW

**Comment by jvickroy** [comment]
Apologies for the _newcomer_ question, but could you point me to a document/discussion describing the motivation for this PR and how themes will be supported going forward?  Thanks.

**Comment by corranwebster** [comment]
Themes have been deprecated for a long time and this decision has had substantial discussion and agreement within Enthought.  In particular they were never available for the Qt backend and there was never any plan to make them available.  We have also been unaware of anyone outside of Enthought using TraitsUI Themes.

There is no intention of supporting TraitsUI themes going forward.

This isn't an irrevocable decision if the community has a compelling use-case that either standard Wx won't solve (or couldn't be solved by Qt Stylesheets), but they do cause significant friction for maintenance of the Wx side of the codebase.

Are you using themed widgets?

**Comment by jvickroy** [comment]
Thanks for your quick response.

I am a traits/traitsui novice, and as such am not currently using themes, but I did intend to in the future.

There is certainly no reason to make maintenance more difficult based on my future _intent_!

I really like traits/traitsui and appreciate them being made publicly-available.  At this point, my major stumbling block is trying to understand the documentation, but that is veering off-topic.

File: traitsui/wx/ui_panel.py [hunk 1]
```diff
@@ -765,17 +726,18 @@ def create_fold_for_items(self, window, content):
         """ Adds a set of groups or items as vertical notebook pages to a
             vertical notebook.
         """
-        from themed_vertical_notebook import ThemedVerticalNotebook
+        raise NotImplementedError('VFold is not implemented for Wx backend')
+        #from themed_vertical_notebook import ThemedVerticalNotebook
```
Comment (mdickinson) [inline traitsui/wx/ui_panel.py]: Please can we either remove this or add an explanation about why the commented-out code is still here?

**Comment by mdickinson** [comment]
LGTM; one nit about some commented-out code.

**Comment by mdickinson** [comment]
LGTM!

**Pull request merged**